tests/dragonfly/replication_test.py

```diff
@@ -2011,11 +2011,11 @@ async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     replica = df_factory.create(dbfilename=f"dump_{tmp_file_name()}")
     df_factory.start_all([master, replica])
 
-    seeder = SeederV2(key_target=40000)
+    seeder = SeederV2(key_target=200000)
```

Comment:
and can we use 2M, instead of 200K - I do not want us to look at this test again in 2months

Reply:
I guess seeder is faster but does it really matter in this context ? :shrug: (what I am trying to say here is I don't have an opinion on this_

Reply:
> https://github.com/dragonflydb/dragonfly/pull/3697#discussion_r1754220572

I can, I just noticed how much the `time increases` in test. On my machine it took 2 seconds and no takes roughly 8. The reason I did not increase it further was because I don't want a small test case like this to take `a minute`. It's up to you, I don't mind increasing the load -- the only concern I had was the time it takes (and I tried to balanced it -- the number I have there was not really "random")

Reply:
I do not think seeder is faster. I think we use it because we use it everywhere. Lets replace it with debug populate with enough data to cause replica to load the data for significant amount of time (like 5s)

Reply:
in release mode

Reply:
Sounds good.

Reply:
takes roughly 6-7 on release on my machine with the new changes